pubsub/client.go

```diff
@@ -122,11 +124,12 @@ const (
 )
 
 // Creates a new Radix client and subscribes to it.
-func New(address string) *Client {
+func New(address, password string) *Client {
```

Comment:
Thats a much neater solution indeed. 
Why is it that Golang needs so many lines just to make a parameter optional? :grin: 
I will push this later today.